packages/dai-plugin-governance/babel.config.js

```diff
@@ -0,0 +1 @@
+../../babel.config.js
```

Comment:
this is a symlink, since the contents would be exactly the same as the file in the root. if this seems to work ok then i'll do the same for the other packages